File: src/main/java/com/graqr/threshr/model/redsky/product/Compliance.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;
import jakarta.annotation.Nullable;

@Serdeable
public record Compliance(
        @Nullable @JsonProperty("is_proposition_65")
        Boolean isProposition65,
        @Nullable @JsonProperty("is_snap_eligible")
        Boolean isSnapEligible) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/Compliance.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable
import jakarta.annotation.Nullable

@Serdeable
data class Compliance(
    @param:Nullable @param:JsonProperty("is_proposition_65") val isProposition65: Boolean,
    @param:Nullable @param:JsonProperty("is_snap_eligible") val isSnapEligible: Boolean
) {}
